entrypoint.sh

```diff
@@ -20,3 +22,5 @@ delimiter=$(openssl rand -hex 20)
 echo "text-diff<<$delimiter" >> $GITHUB_OUTPUT
 echo "$diff" >> $GITHUB_OUTPUT
 echo "$delimiter" >> $GITHUB_OUTPUT
+
+git checkout "$current_head"
```

Comment:
🔧 
We probably should use `git switch` here, right?
```suggestion
git switch "$current_head"
```

Reply:
With `actions/checkout`, it automatically moves to a `detached HEAD` state. To align with this behavior, I chose to use `git checkout`.
However, I agree that `git switch --detach "$current_head"` could also be a viable alternative.